docs/cni/kubernetes/AWS.md

```diff
@@ -199,9 +204,19 @@ wget https://storage.googleapis.com/kubernetes-release/release/v1.1.4/bin/linux/
 chmod +x ./kubectl
 ```
 
-The following command sets up SSH forwarding of port 8080 to your master node so that you can run `kubectl` commands on your local machine. Replace `~/mykeypair.pem` with the location of the keypair you generated earlier, and replace `ec2-###-##-##-###.compute-1.amazonaws.com` with the master public DNS name of your cluster.
+Save the public DNS name for the master in an environment variable. Replace `ec2-###-##-##-###.compute-1.amazonaws.com` with the master public DNS name - you can find this in the AWS portal, or by running `aws ec2 describe-instances`.
```

Comment:
Hmm.  My instances don't have a public DNS name.  Have I done something wrong?  Will attempt to continue using public IP instead.